db/migrate/20220131165431_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name, null: false
```

Comment:
Considerando que name é único, faltou adicionar um index para forçar essa validação a nivel de banco também